packages/react/package.json

```diff
@@ -331,14 +290,13 @@
     "rollup-plugin-filesize": "^9.1.1",
     "rollup-plugin-postcss": "^4.0.0",
     "rollup-plugin-uglify": "^6.0.4",
+    "sass-loader": "^10.0.0",
     "sass": "^1.42.1",
     "storybook-addon-rtl": "^0.2.2",
     "style-loader": "^0.23.1",
-    "stylelint": "^10.1.0",
-    "stylelint-config-recommended-scss": "^3.3.0",
-    "stylelint-scss": "^3.10.0",
+    "stylelint": "^14.0.1",
+    "stylelint-config-standard-scss": "^2.0.1",
     "weak-napi": "^2.0.1",
-    "webpack": "^4.41.5",
```

Comment:
Do we not need this for our cypress test?

Reply:
You're right. We should add this back. I (briefly) toyed with upgrading to webpack 5 and quickly realized that needed to be it's own PR, so this mistakenly got removed.